runtime/onert/frontend/circle_traininfo/src/traininfo_loader.cc

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "traininfo_loader.h"
+#include "circle_traininfo_generated.h"
+#include "flatbuffers/flatbuffers.h"
+
+namespace onert
+{
+namespace train
+{
+namespace traininfo_loader
+{
+
+namespace
+{
+
+ir::train::OptimizerInfo loadOptimizerInfo(const circle::ModelTraining *circle_model)
+{
```

Comment:
Don't you do the null pointer check of `circle_model`? Is this always a valid variable?

Reply:
Thanks for pointing out. 
At least, we could expect that circle_model is not nullptr in this function. 
So, I added an assert statement here. 

Also for you information : https://github.com/Samsung/ONE/pull/12400/files#r1441361872